cassandra-analytics-integration-tests/src/test/java/org/apache/cassandra/analytics/testcontainer/BulkWriteS3CompatModeSimpleMultipleTokensTestImpl.java

```diff
@@ -0,0 +1,12 @@
+package org.apache.cassandra.analytics.testcontainer;
+
+import org.apache.cassandra.testing.ClusterBuilderConfiguration;
+
+public class BulkWriteS3CompatModeSimpleMultipleTokensTestImpl extends BulkWriteS3CompatModeSimpleTest {
```

Comment:
maybe rename to `BulkWriteS3CompatModeSimpleMultipleTokensTest`? The ending `Impl` is probably added by mistake.